productsPage/products.html

```diff
@@ -6,9 +6,59 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="products.css" />
     <title>Products</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+   <!-- icon Library -->
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+
 </head>
 <body>
-    
+    <section class="continar-all-page">
+              <!--  *************** Start  Header  ************* -->
+
+        <!-- header for mobile and ipad  -->
+        <header>
+       <a href="/index.html"> <i class="ri-arrow-left-s-line"></i></a>
+          <form class="search-form">
+            <img src ="/images/Search.svg"class = "search-icon" alt="" />
+            <input
+              type="text"
+              placeholder="    Search"
+              class="search-input"
+            />
+          </form>
+      <!-- header element for desktop-->
+      <nav >
+        <li>   <a href="/index.html"  >Home </a>  </li>
+        <li>    <a href="/cartPage/cart.html"  > My Cart</a> </li>
+        <li>    <a href="/aboutPage/about.html" >About Us </a> </li>
```

Comment:
the links will not work in GitHub pages with this path, you need to link it directly 

```suggestion
<li>   <a href="../index.html"  >Home </a>  </li>
        <li>    <a href="../cartPage/cart.html"  > My Cart</a> </li>
        <li>    <a href="../cartPage/cart.html" >About Us </a> </li>
```

Reply:
done